### PR TITLE
第一版翻译完成

### PR DESCRIPTION
**Pull request by thanksdanny**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by thanksdanny** [comment]
@lihuazhang ok

File: docs/cn/appium-setup/android-setup.md [hunk 1]
```diff
@@ -1,49 +1,41 @@
...
+开始配置之前，你需要安装 Node.js（v4 或更高版本）。具体请参考 [instructions for your flavor of linux](https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager)。
 
-Once you've got Node.js installed, install the [Android SDK](http://developer.android.com/sdk/index.html).
-You will need to run the `android` tool (included in the SDK, under the 'tools' directory).
+在你安装完 Node.js 后，下一步就是安装 [Android SDK](http://developer.android.com/sdk/index.html).
+因为你需要运行  `android` tool（包含 SDK， 在 'tools' 目录下）。
 
-Run the `android` tool and use it to install an API Level 17 or greater.
 
-(If you want to run Appium from source, you will also need [Apache Ant](http://ant.apache.org/) to build the bootstrap jar that Appium uses for running against Android simulators/devices.)
```
Comment (thanksdanny) [inline docs/cn/appium-setup/android-setup.md]: @hppyhjh 感谢！这句确实翻译得不妥，尤其是 against 这里。你的话这句会如何翻译呢？

Comment (xiaozi0lei) [inline docs/cn/appium-setup/android-setup.md]: @thanksdanny 翻译如下：`如果想通过源码运行 Appium，需要先用 Apache Ant 构建出 bootstrap jar 包，Appium 将依赖该 jar 包在 Android 模拟器或真机上运行。`

不知翻译如何，供参考。

Comment (thanksdanny) [inline docs/cn/appium-setup/android-setup.md]: 感谢！已参考！

**Comment by thanksdanny** [comment]
@lihuazhang @xiaozi0lei @hppyhjh 已经根据审核同学的修改建议进行修改，修改后提交了新commit，[传送门](https://github.com/testerhome/appium/pull/10)

**Comment by xiaozi0lei** [comment]
@thanksdanny LGTM. 👍

**Pull request merged**